tests/python/pytorch/graphbolt/test_dataset.py

```diff
@@ -14,3 +18,36 @@ def test_Dataset():
         _ = dataset.graph()
     with pytest.raises(NotImplementedError):
         _ = dataset.feature()
+
+
+def test_OnDiskDataset_TVTSet():
```

Comment:
You plan to implement train_set e.t.c. in the future PR, right?
This actually does not test _TVTSet, just initialization and metadata validation.
```suggestion
def test_OnDiskDataset_Initialization():
```

Reply:
yes, I'd like to parse TVT in several PRs so I use the name `test_xxx_TVTSet()`. As for the suggested name `Inialization`, it indicates init the whole dataset for me. But actually, I'd like to test init TVT.